contrib/docker/Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM docker.io/golang:alpine as builder
+FROM golang:alpine as builder
 
-COPY . /src
 WORKDIR /src
 
 ENV CGO_ENABLED=0
 
-RUN apk add git && ./build 
+RUN apk add git && \
+    git clone https://github.com/yggdrasil-network/yggdrasil-go.git . && \
+    ./build
```

Comment:
*Why?*
It makes sure that you can only ever build the latest master commit and nothing else. This is horrible.